django_project/changes/templates/entry/includes/entry_detail.html

```diff
@@ -17,6 +17,7 @@ <h3><span class="text-muted">Feature:</span> {{ entry.title }}</h3>
                         <span class="glyphicon glyphicon-thumbs-up"></span>
                     </a>
                 {% endif %}
+                {% if user_can_cud %}
```

Comment:
I think it would be nice if we setup always `user_can_edit` `user_can_delete`, `user_can_create` or whatever

Because every logged users can CRU, but only the owner (or changelog manager, or staff) can Delete

But maybe stay focus on `Version` and not `Entry` in this PR

Reply:
Updated